src/components/tabset/tabset.component.html

```diff
@@ -8,10 +8,10 @@
     [attr.aria-label]="ariaLabel"
     [attr.aria-orientation]="stacked === 'none' ? 'horizontal' : 'vertical'">
 
-	<li role="presentation"
+    <li role="presentation"
         class="nav-item"
         *ngFor="let tab of _tabset.tabs; let index = index"
-        [class.active]="tab.active"
+        [class.active]="(_tabset.activeTab$ | async) === tab"
```

Comment:
I just noticed one thing regarding focus. If i programmatically set the active tab, the focusable item remains on the previously selected tab.

Eg. Select the 4th tab, then click the button to activate the second tab, then use the keyboard to return focus to the tabset. The 4th tab will regain focus:

![image](https://user-images.githubusercontent.com/20795331/81287560-e1ebba00-905a-11ea-913b-d91b0f95f846.png)

Reply:
I'll take a look but I think this is a pre-existing bug.

Reply:
Ah right, no problem. In that case feel free to raise a ticket and we can deal with it another time